src/helpers/FileHelpers.ts

```diff
@@ -64,4 +64,18 @@ export class FileHelpers {
       }
     });
   }
+
+  static async getConfigFromFile(configPath: string): Promise<any> {
```

Comment:
You have to write some unit tests here 🙂 

Make sure you test error states as well:
- What if the file exist but is not valid JS?
- What if the file exist but is TypeScript?
- What if the file exist and is correct JS, but does not expose the right config?
- What if the file exist and is correct JS, but does not expose the config file the right way?

Reply:
Make sure you document this function as well i.e. what it does 🙂

Reply:
@jonaslagoni 
I'll make sure to document and write the unit tests.

> What if the file exist and is correct JS, but does not expose the right config?

and 

> What if the file exist and is correct JS, but does not expose the config file the right way?

Should be the same thing right ? I mean we wouldn't know if the file is exposing the config the right way, unless we know what it's exposing ? :thinking: 

The way i put it here might be confusing, feel free to ignore this, i will let you know later more context on this. :slightly_smiling_face:

Reply:
I'll try fixing things one after other and contact you back :smile:

Reply:
@jonaslagoni 

We can restrict the users to only allow them to use js file for config right ? i hardly saw places where people used ts files for config.
What do you say ?

Reply:
Agree 👍 

It can also be an improvement down the line if ever requested 🙂